test/Microsoft.AspNetCore.Mvc.RazorPages.Test/Infrastructure/PageDirectiveFeatureTest.cs

```diff
@@ -48,7 +48,7 @@ public void TryGetPageDirective_JunkBeforeDirective()
             Assert.Null(template);
         }
 
-        [Fact]
+        [Fact(Skip = "This fails")]
```

Comment:
Yeah this should either be a negative test or just deleted. We're not going to support weird garbage stuff.